pkg/ui/v1alpha3/backend.go

```diff
@@ -80,31 +61,31 @@ func (k *KatibUIHandler) FetchHPJobs(w http.ResponseWriter, r *http.Request) {
 
 }
 
+// FetchAllHPJobs gets experiments in all namespaces.
+func (k *KatibUIHandler) FetchAllHPJobs(w http.ResponseWriter, r *http.Request) {
+	// Use "" to get experiments in all namespaces.
+	jobs, err := k.getExperimentList("")
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+	log.Printf("?????: %v", jobs)
```

Comment:
????? by mistake?

Reply:
It is still WIP, some bugs are not fixed. Thus it is a WIP log.

Reply:
Missed WIP tag, please go ahead